app/controllers/v0/ppiu_controller.rb

```diff
@@ -10,10 +10,29 @@ def index
              serializer: PPIUSerializer
     end
 
+    def update
+      pay_info = EVSS::PPIU::PaymentAccount.new(
+        params.permit(ppiu_params)
```

Comment:
I believe you should just be able to call `EVSS::PPIU::PaymentAccount.new(ppiu_params)`, or no?

Reply:
Yes >_< apparently I didnt rerun my tests to catch this